Trailheads/src/classes/AnimalLocator.cls

```diff
@@ -0,0 +1,31 @@
+public with sharing class AnimalLocator {
+
+    public static String getAnimalNameById(Integer animalId) {
+        Http http = new Http();
+        HttpRequest request = new HttpRequest();
+        request.setEndpoint('https://th-apex-http-callout.herokuapp.com/animals/' + animalId);
```

Comment:
String chociaż nad metodę.
Pozwolę Ci tak zostawiać, ale przy każdym takim czymś ba być komentarzu w stylu:
// TODO w kodzie produkcyjnym to bym dał w custom labelce / settingu / klasie ze staticami